lib/components/Notification/notification.scss

```diff
@@ -100,13 +100,14 @@
     &__banners--mobile {
         position: absolute;
         width: 360px;
-        bottom: var(--semantic-spacing-general-sm);
-        left: var(--semantic-spacing-general-sm);
+        inset-block-end: var(--semantic-spacing-general-sm);
+        inset-inline-start: var(--semantic-spacing-general-sm);
         z-index: 6;
 
         &--mobile {
-            top: var(--semantic-spacing-general-sm);
-            bottom: unset;
+            width: calc(100% - var(--semantic-spacing-general-md));
+            inset-block-start: var(--semantic-spacing-general-sm);
+            inset-block-end: unset;
```

Comment:
using CSS logical properties instead of physical ones (top/left/bottom/etc.) to make sure it supports RTL in future